### DISCUSSION TITLE
Version 1.30.1

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#5639] [Changelog]
## Bugfixes
- [#5246](https://github.com/ScratchAddons/ScratchAddons/issues/5246) scratch-messaging always shows time in English. (prio-4)
- [#5508](https://github.com/ScratchAddons/ScratchAddons/issues/5508) clones show a negative counter value. (prio-4)
- [#5560](https://github.com/ScratchAddons/ScratchAddons/issues/5560) vol-slider shows a wrong default volume. (prio-4)
- [#5562](https://github.com/ScratchAddons/ScratchAddons/issues/5562) gamepad loses the control settings after unplugging. (prio-3)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties breaks in some locales. (prio-2)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties disappears when changing locales. (prio-2)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties does not remember whether the panel is open. (prio-3)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties info button disappears after renaming a sprite. (prio-3)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties has a performance issue. (prio-3)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties is incompatible with folders. (prio-3)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties collapse button is draggable. (prio-4)
- [#5563](https://github.com/ScratchAddons/ScratchAddons/issues/5563) sprite-properties panel can be hidden momentarily in some cases. (prio-4)
- [#5566](https://github.com/ScratchAddons/ScratchAddons/issues/5566) variable-manager crashes when displaying large variable values. (prio-2)
- [#5570](https://github.com/ScratchAddons/ScratchAddons/issues/5570) drag-drop is incompatible with sprite-properties. (prio-3)
- [#5584](https://github.com/ScratchAddons/ScratchAddons/issues/5584) Addons do not run in `/projects` error page. (prio-4)
- [#5585](https://github.com/ScratchAddons/ScratchAddons/issues/5585) paint-snap breaks the reshape tool. (prio-2)
- [#5586](https://github.com/ScratchAddons/ScratchAddons/issues/5586) zebra-stripping does not distinguish between blocks of the same color. (prio-4)
- [#5588](https://github.com/ScratchAddons/ScratchAddons/issues/5588) editor-searchable-dropdowns allow creation of conflicting variables. (prio-3)
- [#5588](https://github.com/ScratchAddons/ScratchAddons/issues/5588) editor-searchable-dropdowns does not trim the variable names. (prio-4)
- [#5592](https://github.com/ScratchAddons/ScratchAddons/issues/5592) Addon block insertion markers are colored, unlike vanilla blocks. (prio-4)
- [#5593](https://github.com/ScratchAddons/ScratchAddons/issues/5593) Some addon modals have wrong animation. (prio-4)
- [#5594](https://github.com/ScratchAddons/ScratchAddons/issues/5594) Addon settings show "Presets" next to, not above, preset buttons. (prio-4)
- [#5597](https://github.com/ScratchAddons/ScratchAddons/issues/5597) scratch-messaging does not wrap long studio and topic names. (prio-4)
- [#5600](https://github.com/ScratchAddons/ScratchAddons/issues/5600) middle-click-popup has a wrong description. (prio-4)
- [#5622](https://github.com/ScratchAddons/ScratchAddons/issues/5622) fullscreen makes the canvas white. (prio-2)
- [#5624](https://github.com/ScratchAddons/ScratchAddons/issues/5624) Setting reset dropdown appears when there is no preset to reset to. (prio-4)
- [#5625](https://github.com/ScratchAddons/ScratchAddons/issues/5625) opacity-slider does not allow clicking to change values. (prio-3)

## Changes
- [#5540](https://github.com/ScratchAddons/ScratchAddons/issues/5540) The speed of "open in full screen" hover in popup setting is changed. (prio-4)
- [#5583](https://github.com/ScratchAddons/ScratchAddons/issues/5583) progress-bar: change the bar design (prio-4)

**Comment by DNin01** [comment]
I was hoping to see some of the recent bug fixes and small improvements we've made soon, and here they are! 🎉

**Comment by BroJac5246** [comment]
Should #5628 be included?

**Reply by DNin01** [reply]
That's another part of the same change for the same update, so I think it's kind of already included.

**Reply by BroJac5246** [reply]
> That's another part of the same change for the same update, so I think it's kind of already included.

It should at least be directly mentioned, though.

**Reply by DNin01** [reply]
Like `- #5583 and #5628 progress-bar: change the bar design (prio-4)`?
